yt_dlp/extractor/newgrounds.py

```diff
@@ -202,7 +213,57 @@ def _real_extract(self, url):
                 continue
             entries.append(
                 self.url_result(
-                    'https://www.newgrounds.com/%s' % path,
+                    f'https://www.newgrounds.com/{path}',
                     ie=NewgroundsIE.ie_key(), video_id=media_id))
 
         return self.playlist_result(entries, playlist_id, title)
+
+
+class NewgroundsUserIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<id>[^\.]+)\.newgrounds\.com/(movies|audio)'
+    IE_NAME = 'Newgrounds:user'
+    _PAGE_SIZE = 30
+    _TESTS = [{
+        'url': 'https://burn7.newgrounds.com/audio',
+        'info_dict': {
+            'id': 'burn7',
+        },
+        'playlist_mincount': 150,
+    }, {
+        'url': 'https://burn7.newgrounds.com/movies',
+        'info_dict': {
+            'id': 'burn7',
+        },
+        'playlist_mincount': 2,
+    }, {
+        'url': 'https://brian-beaton.newgrounds.com/movies',
+        'info_dict': {
+            'id': 'brian-beaton',
+        },
+        'playlist_mincount': 10,
+    }]
+
+    def _fetch_page(self, channel_id, url, page):
+        page += 1
+        posts_info = self._download_json(
+            f'{url}/page/{page}', channel_id,
+            note=f'Downloading page {page}', errnote=f'Failed to downloading page {page}', headers={
```

Comment:
```suggestion
            note=f'Downloading page {page}', errnote=f'Failed to download page {page}', headers={
```
though the errnote is unnecessary here imo